test/k8s-integration/prepull-image.sh

```diff
@@ -4,22 +4,40 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-if [[ -z "${PREPULL_IMAGE}" ]]; then
+# This is taken from prepull.yaml.
+readonly prepull_daemonset=prepull-test-containers
+
+wait_on_prepull()
+{
+    # Wait up to 15 minutes for the test images to be pulled onto the nodes.
+    retries=90
+    while [[ $retries -ge 0 ]];do
+        ready=$(kubectl get daemonset "${prepull_daemonset}" -o jsonpath="{.status.numberReady}")
```

Comment:
this logic is similar to the other wait script, maybe we can consolidate to a common function later?

Reply:
I think we should probably move to client-go and have it in code. That's a bigger change than I want to do right now though.

Consolidating would mean have a shared bash include directory between deploy/kubernetes and test/k8s-integration which starts to get messy.

Reply:
#625 created for the client-go work.